plugins/engines/ospray/OSPRayEngine.cpp

```diff
@@ -290,6 +290,14 @@ void OSPRayEngine::_createRenderers()
                 {"shadows", "Shadow intensity", 0.f, {0.f, 1.f}});
             properties.setProperty(
                 {"softShadows", "Shadow softness", 0.f, {0.f, 0.1f}});
+            properties.setProperty({"samplingThreshold",
+                                    "Threshold under which sampling is ignored",
+                                    0.01f,
+                                    {0.01f, 1.f}});
+            properties.setProperty({"volumeSpecularExponent",
+                                    "Volume specular exponent",
+                                    1.f,
```

Comment:
default value is 20 in osp?

Reply:
yes, 'ns' attribute in ospray/volume/volume.cpp

Reply:
? here it's 1.f, but here it's 20: https://github.com/BlueBrain/Brayns/pull/484/files#diff-cdd02215d26908726f81514dace27955R63